Kernel/VirtIO/VirtIOQueue.cpp

```diff
@@ -103,6 +72,26 @@ void* VirtIOQueue::get_buffer(size_t* size)
     return token;
 }
 
+FlatPtr VirtIOQueue::get_buffer_paddr(size_t *size)
```

Comment:
does size need to be a pointer? can it be a reference?

Reply:
Fixed in new revision